packages/scheduler/src/Scheduler.js

```diff
@@ -649,6 +647,19 @@ if (typeof window !== 'undefined' && window._schedMock) {
     }
   };
 
+  if (typeof MessageChannel === 'function') {
+    // Use a MessageChannel, if support exists
+    var channel = new MessageChannel();
+    channel.port1.onmessage = idleTick;
+    port = channel.port2;
+  } else {
+    // Otherwise post a message to the window. This isn't ideal because message
+    // handlers will fire on every frame until the queue is empty, including
+    // some browser extensions.
```

Comment:
Do we need this fallback? IE9? Do we even support that anyway?

Reply:
Idk, for some reason I thought FB did but I guess we don't any longer